tests/test-cases/cli/inspect/itis-industry-multiple-skos-inscheme.csv-metadata2.json

```diff
@@ -0,0 +1,124 @@
+{
```

Comment:
Can we give this a better title which indicates what is different/why we need it

Reply:
done